src/main/resources/edu/cornell/kfs/ksr/businessobject/datadictionary/SecurityGroupTab.xml

```diff
@@ -0,0 +1,171 @@
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="40" />
+		</property>
+	</bean>
+		
+	<bean id="SecurityGroupTab-tabOrder" parent="SecurityGroupTab-tabOrder-parentBean" />
+	<bean id="SecurityGroupTab-tabOrder-parentBean" parent="AttributeDefinition">
+		<property name="name" value="tabOrder" />
+		<property name="label" value="Tab Order" />
+		<property name="shortLabel" value="Order" />
+		<property name="maxLength" value="2" />
+		<property name="required" value="true" />
+		<property name="summary" value="Tab Order" />
+		<property name="description" value="Tab Order" />
+		<property name="validationPattern">
+			<bean parent="NumericValidationPattern" />
+		</property>
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="2" />
+		</property>
+	</bean>
+
+	<bean id="SecurityGroupTab-active" parent="SecurityGroupTab-active-parentBean" />
+	<bean id="SecurityGroupTab-active-parentBean" abstract="true" parent="AttributeReferenceDummy-activeIndicator">
+		<property name="name" value="active" />
+	</bean>
+
+	<bean id="SecurityGroupTab-versionNumber" parent="SecurityGroupTab-versionNumber-parentBean" />
+	<bean id="SecurityGroupTab-versionNumber-parentBean" abstract="true" parent="AttributeReferenceDummy-versionNumber"></bean>
+	
+	<!-- Business Object Inquiry Definition -->
+
+    <bean id="SecurityGroupTab-inquiryDefinition" parent="SecurityGroupTab-inquiryDefinition-parentBean"/>
+    <bean id="SecurityGroupTab-inquiryDefinition-parentBean" abstract="true" parent="InquiryDefinition"
+          p:title="Security Group Tab Inquiry" >
+        <property name="inquirySections">
+            <list>
+                <ref bean="SecurityGroupTab-inquirySectionDefinition"/>
+            </list>
+        </property>
+    </bean>
+    
+    <bean id="SecurityGroupTab-inquirySectionDefinition" parent="SecurityGroupTab-inquirySectionDefinition-parentBean"/>
+    <bean id="SecurityGroupTab-inquirySectionDefinition-parentBean" parent="InquirySectionDefinition" abstract="true"
+          p:title="">
+        <property name="inquiryFields">
+            <list>
+                <bean parent="FieldDefinition" p:attributeName="securityGroupId"/>
+                <bean parent="FieldDefinition" p:attributeName="tabId"/>
+                <bean parent="FieldDefinition" p:attributeName="tabName"/>
+                <bean parent="FieldDefinition" p:attributeName="tabOrder"/>
+                <bean parent="FieldDefinition" p:attributeName="active"/>
+            </list>
+        </property>
+    </bean>
+    
+    <!-- Business Object Lookup Definition -->
+
+    <bean id="SecurityGroupTab-lookupDefinition" parent="SecurityGroupTab-lookupDefinition-parentBean"/>
+    <bean id="SecurityGroupTab-lookupDefinition-parentBean" abstract="true"
+          parent="SecurityGroupTab-lookupDefinition-base-parentBean"/>
+    <bean id="SecurityGroupTab-lookupDefinition-base-parentBean" abstract="true" parent="LookupDefinition"
```

Comment:
Another bean that might not need the extra parent level.

Reply:
updated